controllers/sonataflow_controller_test.go

```diff
@@ -58,7 +58,7 @@ func TestKogitoServerlessWorkflowController(t *testing.T) {
 		if err != nil {
 			t.Fatalf("reconcile: (%v)", err)
 		}
-		afterReconcileWorkflow := &v1alpha08.KogitoServerlessWorkflow{}
+		afterReconcileWorkflow := &v1alpha08.SonataFlow{}
 		if err := cl.Get(context.TODO(), req.NamespacedName, afterReconcileWorkflow); err != nil {
 			t.Fatalf("Failed to fetch supposed to exist workflow %v", err)
```

Comment:
```suggestion
			t.Fatalf("Failed to fetch workflow %v", err)
```

Reply:
@spolti I won't change anything not related to the renaming in this PR.

Reply:
Because we can chery-pick/link back to it later if needed, and the only work was related to the renaming.